docs/design/api/storage.rst

```diff
@@ -0,0 +1,324 @@
+Storage API
+###########
+
+For this examples let's say:
+
+- Buyers logs with Firefox Account Bearer Tokens
+- Seller apps logs with Hawk
+- Payment App logs with Basic Auth
+
+
+As a user, I want to retrieve my information about my purchases
+===============================================================
+
+.. code-block:: http
+
+    GET /buckets/mozilla/collections/payments/records
+    Authorization: Bearer <User Firefox Account Bearer Token>
```

Comment:
Thanks a lot for this. I was looking for it for a long time. I will change it tomorrow.